src/dispatchers/networkDispatchers.ts

```diff
@@ -214,3 +216,12 @@ export class FetchRequestDispatcher extends Dispatcher<FetchRequest, channels.Fe
   }
 }
 
+function deserializeFormData(formData?: channels.FormField[]): FormField[] | undefined {
+  if (!formData)
+    return undefined;
+  for (const field of formData) {
+    if (field.file)
+      (field.file.buffer as any) = Buffer.from(field.file.buffer, 'base64');
```

Comment:
I'd suggest to create a new object here, just in case the old one is used somewhere in dispatcher/tracing/instrumentation.

Reply:
Removed this code.